README.md

```diff
@@ -51,32 +34,34 @@ pip install -e .
 ### Common issues
 
 You may end up with a version of Keras incompatible with the marshalled
-data in `pisces/cached_models`. In that case, re-run the generation
-notebook `<pisces>/analyses/convert_mads_olsen_model_to_keras.ipynb`
+data in `pisces/cached_models`. In that case, run
+[`pisces_setup`](https://Arcascope.github.io/pisces/mads_olsen_support.html#pisces_setup)
+in a terminal;
+[`pisces_setup`](https://Arcascope.github.io/pisces/mads_olsen_support.html#pisces_setup)
+is in your path as long as a Python environment with `pisces` installed
+is active.
 
 ## Usage
 
-The primary module to import is `pisces.experiments` which contains
-classes used for discovering and providing access to data sets in your
-chosen folder, as well as (trainable) classifiers.
+The pipeline is intended to be flexible and can be easily extended to
+include new models, datasets, and evaluation metrics. In version 2.0, we
```

Comment:
Propose changing to `In version 2.0, we have streamlined the library to prioritizing nimbleness and easy debugging.`